atomic_reactor/inner.py

```diff
@@ -496,6 +493,10 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
+            buildstep_runner = BuildStepPluginsRunner(self.builder.tasker, self,
```

Comment:
Can we add a comment to explain why we delay creating this?

Reply:
added